### ISSUE TITLE
SIGSEGV on 1.3.3

### ISSUE THREAD

**Issue by dharrigan** [clojure-goes-fast/clj-async-profiler#40]
Hi,

Getting a core dump using 1.3.3.

```
#
# A fatal error has been detected by the Java Runtime Environment:
#
#  SIGSEGV (0xb) at pc=0x000077e67da078af, pid=236623, tid=236862
#
# JRE version: OpenJDK Runtime Environment Temurin-23.0.1+11 (23.0.1+11) (build 23.0.1+11)
# Java VM: OpenJDK 64-Bit Server VM Temurin-23.0.1+11 (23.0.1+11, mixed mode, sharing, tiered, compressed oops, compressed class ptrs, g1 gc, linux-amd64)
# Problematic frame:
# C  [libasyncProfiler-linux-x64.so+0x78af]  NMethod::isNMethod()+0x1f
#
# Core dump will be written. Default location: Core dumps may be processed with "/usr/lib/systemd/systemd-coredump %P %u %g %s %t %c %h" (or dumping to /home/david/development/foo/bar/core.236623)
#
# An error report file with more information is saved as:
# /home/david/development/foo/bar/hs_err_pid236623.log
[30.172s][warning][os] Loading hsdis library failed
#
# If you would like to submit a bug report, please visit:
#   https://github.com/adoptium/adoptium-support/issues
#```

Happy to share the dump, but in a more private setup.

-=david=-

**Comment by alexander-yakushev** [comment]
Hello! Are you sure you use 1.3.3 and not some older version pulled transitively? This precise issue should be fixed in 1.3.3. Can you check the final classpath?

**Comment by dharrigan** [comment]
Hi,

Had time to retest using 1.4.0 (and Temurin-23.0.1+11). Unable to reproduce. Closing this issue! (thanks for a great tool btw! :-) )
